packages/api/tests/lib/test-orca.js

```diff
@@ -96,10 +117,17 @@ test.serial(
   'getOrcaRecoveryStatusByGranuleId returns running status when file restore has error',
```

Comment:
```suggestion
  'getOrcaRecoveryStatusByGranuleId returns failed status when file restore has error',
```
I think this should be changed.